### ISSUE TITLE
[FR] Auto switch light/dark theme

### ISSUE THREAD

**Issue by cxyzzz** [k0shk0sh/FastHub#2885]
**Is your feature request related to a problem? Please describe.**

I need to switch to light mode during the day and dark mode at night, which is very troublesome. I hope that the application can automatically switch with the system.

**Describe the solution you'd like**

The application theme can automatically switch with the system.

**Comment by ushashwat** [comment]
Yes, this would be helpful. Or maybe switching themes according to Android 10's system default theme.

**Comment by Yasserboussaden** [comment]
Yasserboussaden@gmail.com
